spec/set.js

```diff
@@ -93,17 +92,26 @@ describe.only('Set', () => {
   context('union()', () => {
     it('Adds \'A\',\'B\',\'C\' then returns true if union [\'D\',\'E\',\'F\'] returns a set with size of 6', () => {
       const mySet = new Set()
+      const otherSet = new Set()
       mySet.add('A')
       mySet.add('B')
       mySet.add('C')
-      expect(mySet.union(['D','E','F']).length).to.equal(6)
+      otherSet.add('D')
+      otherSet.add('E')
+      otherSet.add('F')
+      expect(mySet.union(otherSet).length).to.equal(6)
     })
     it('Adds \'A\',\'B\',\'C\' then returns true if union [\'C\',\'D\',\'E\',\'F\'] returns the set [\'A\',\'B\',\'C\',\'D\',\'E\',\'F\']', () => {
```

Comment:
You can avoid the use of all of these backslashes by replacing the outer single quotes with a double quote.  This would definitely increase the readability of these tests:
```js
it( "Adds 'A', 'B', 'C'" /* etc */ ) {
```
The description of this test is also confusing to me.  The description suggests that characters are being added and a boolean value is being returned, but the test should really be testing that two distinct`Set` objects are being `union`ed, with a new `Set` resulting from the union.